pkg/tools/after/func.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2021 Doc.ai and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package after provides wrapper for the `time.AfterFunc` with some extensions.
+package after
```

Comment:
I'm not sure that we need to implement this.

1. Why serialize. executor is required for it?
2. `after.NewFunc()` looks not clear.
3. Why API Resume is required?

Potentially cancelable timer is already implemented by go community, please let's investigate it.